Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Create_a_security_pre-rule_for_EDL.yml

```diff
@@ -0,0 +1,610 @@
+        simple: any
+      Destination:
+        complex:
+          root: PrismaSase.ExternalDynamicList
+          accessor: name
+      Folder:
+        complex:
+          root: inputs.Folder
+      Overwrite:
+        simple: "False"
+      Position:
+        simple: pre
+      RuleName:
+        complex:
+          root: inputs.RuleName
+      Service:
+        simple: any
+      Source:
+        simple: any
+      TSGID:
+        complex:
+          root: inputs.TSGID
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 0
+    view: |-
+      {
+        "position": {
+          "x": 940,
+          "y": 550
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "13":
+    id: "13"
+    taskid: 166e379f-9232-48ec-81ed-e1570c467865
+    type: condition
+    task:
+      id: 166e379f-9232-48ec-81ed-e1570c467865
+      version: -1
+      name: Complete this task to push the configuration.
+      description: Complete this task to push the configuration.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "4"
+      Push:
+      - "1"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 560,
+          "y": 910
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+      replyOptions:
+      - Push
+      - "No"
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+system: true
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 1675,
+        "width": 1000,
+        "x": 320,
+        "y": -350
+      }
+    }
+  }
+inputs:
+- key: TSGID
+  value: {}
+  required: false
+  description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+  playbookInputQuery:
+- key: Folder
+  value:
+    simple: Shared
+  required: false
+  description: |-
+    The configuration folder group setting.
+    The default value is 'Shared'.
+  playbookInputQuery:
+- key: URL
+  value: {}
+  required: false
+  description: The source URL. If the type is ip, url or domain, a value must be provided.
+  playbookInputQuery:
+- key: EDLObjectName
+  value: {}
+  required: false
+  description: The external dynamic list object name.
+  playbookInputQuery:
+- key: RuleName
+  value:
+    simple: 'Block Indicators using EDL - XSOAR incident #${incident.id}'
```

Comment:
If we adding the incident ID in the rule name - we will always create a new one since the incident ID changes consistently. lets remove it from the name

Reply:
But that's the idea - force them to use the address group object instead of the security rule.
If you want - I can add another flow -> there the user will specify the Rule name that needs to be edited.

Reply:
You cant force them to do anything lol. if they want to add a security rule, they can.
If the input of the security rule empty - lets go to the flow you suggested :) liked it